app/src/main/java/ru/thstdio/aa2020/ui/detail/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,92 @@
+import by.kirich1409.viewbindingdelegate.viewBinding
+import coil.load
+import coil.transform.BlurTransformation
+import kotlinx.coroutines.*
+import ru.thstdio.aa2020.R
+import ru.thstdio.aa2020.api.loadMovie
+import ru.thstdio.aa2020.data.Movie
+import ru.thstdio.aa2020.databinding.FragmentMoviesDetailsBinding
+import ru.thstdio.aa2020.ui.FragmentNavigation
+
+
+class FragmentMoviesDetails : FragmentNavigation(R.layout.fragment_movies_details) {
+    companion object {
+        private const val CinemaArg = "Cinema_ID"
+        fun newInstance(cinema: Long): FragmentMoviesDetails {
+            val fragment = FragmentMoviesDetails()
+            val bundle = Bundle(1)
+            bundle.putLong(CinemaArg, cinema)
+            fragment.arguments = bundle
+            return fragment
+        }
+    }
+
+    private val binding: FragmentMoviesDetailsBinding by viewBinding()
+    private val exceptionHandler = CoroutineExceptionHandler { coroutineContext, exception ->
+        println("CoroutineExceptionHandler got $exception in $coroutineContext")
+    }
+    private val scope: CoroutineScope = CoroutineScope(
+        Dispatchers.Main.immediate +
+                exceptionHandler
+    )
+
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.areaBack.setOnClickListener {
+            router.back()
+        }
+        arguments?.let { arg ->
+            val cinemaId = arg.getLong(CinemaArg)
+            scope.launch {
+
+                val cinema = loadMovie(requireContext(), cinemaId)
+                withContext(Dispatchers.Main) { bindView(cinema) }
+            }
+        }
+    }
+
+    private fun bindView(cinema: Movie) {
+        binding.textTitle.text = cinema.title
+        setBg(cinema.backdrop)
+        binding.textTag.text = cinema.genres.joinToString { it.name }
+        binding.textReviews.text = getString(R.string.review_string, cinema.numberOfRatings)
+        binding.textStory.text = cinema.overview
+        binding.rating.setRating(cinema.ratings)
+        if (cinema.actors.isNotEmpty()) {
+            binding.recyclerView.layoutManager =
+                LinearLayoutManager(requireContext(), RecyclerView.HORIZONTAL, false)
+            val adapter = ActorAdapter()
+            adapter.setActors(cinema.actors)
+            binding.recyclerView.adapter = adapter
+        } else {
+            binding.textCast.isVisible = false
+            binding.recyclerView.isVisible = false
+        }
+    }
+
+    private fun setBg(poster: String) {
+        binding.imageBgOrig.load(poster) {
+            crossfade(true)
+            placeholder(R.drawable.cinema_holder)
+            transformations(
+                listOf(
+                    //GrayscaleTransformation(),
+                    BlurTransformation(requireContext(), 5f)
+                )
+            )
+        }
+    }
+
+    override fun onDestroyView() {
```

Comment:
onDestroyView can be called more than once 
u should control it with creating scope inside onViewCreated